opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -188,6 +188,15 @@ message Metric {
     ExponentialHistogram exponential_histogram = 10;
     Summary summary = 11;
   }
+
+  // Additional metadata attributes that describe the metric. [Optional].
+  // Attributes are non-identifying.
+  // Consumers SHOULD not need to be aware of these attributes.
+  // These attributes MAY be used to encode information allowing
+  // for lossless roundtrip translation to / from another data model.
+  // Attribute keys MUST be unique (it is not allowed to have more than one
+  // attribute with the same key).
+  repeated opentelemetry.proto.common.v1.KeyValue metadata_attributes = 12;
```

Comment:
To avoid potential confusion about misinterpreting these with attributes, how about:
```suggestion
  repeated opentelemetry.proto.common.v1.KeyValue metadata = 12;
```
or 
```suggestion
  repeated opentelemetry.proto.common.v1.KeyValue properties = 12;
```